src/main/java/com/hierynomus/mssmb2/messages/SMB2Flush.java

```diff
@@ -32,8 +32,9 @@ public SMB2Flush() {
         super();
     }
 
-    public SMB2Flush(SMB2Dialect smbDialect) {
+    public SMB2Flush(SMB2Dialect smbDialect, SMB2FileId fileId) {
         super(24, smbDialect, SMB2MessageCommandCode.SMB2_FLUSH);
+        this.fileId = fileId;
```

Comment:
That's up to you. I don't think it's useful anymore, but I left it since it was already there.